calico-enterprise_versioned_docs/version-3.18/releases.json

```diff
@@ -1,259 +1,263 @@
         "image": "tigera/compliance-reporter",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "compliance-snapshotter": {
         "image": "tigera/compliance-snapshotter",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "compliance-server": {
         "image": "tigera/compliance-server",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "compliance-benchmarker": {
         "image": "tigera/compliance-benchmarker",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "ingress-collector": {
         "image": "tigera/ingress-collector",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "l7-collector": {
         "image": "tigera/l7-collector",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "license-agent": {
         "image": "tigera/license-agent",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "linseed": {
         "image": "tigera/linseed",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "tigera-cni": {
         "image": "tigera/cni",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "firewall-integration": {
         "image": "tigera/firewall-integration",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "egress-gateway": {
         "image": "tigera/egress-gateway",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "honeypod": {
         "image": "tigera/honeypod",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "honeypod-exp-service": {
         "image": "tigera/honeypod-exp-service",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "honeypod-controller": {
         "image": "tigera/honeypod-controller",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "key-cert-provisioner": {
         "image": "tigera/key-cert-provisioner",
-        "version": "master",
+        "version": "v1.1.10",
         "registry": "quay.io"
       },
       "anomaly_detection_jobs": {
         "image": "tigera/anomaly_detection_jobs",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "anomaly-detection-api": {
         "image": "tigera/anomaly-detection-api",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "elasticsearch-metrics": {
         "image": "tigera/elasticsearch-metrics",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "packetcapture": {
         "image": "tigera/packetcapture",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "prometheus": {
         "image": "tigera/prometheus",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "coreos-prometheus": {
-        "version": "v2.42.0"
+        "version": "v2.43.1"
```

Comment:
This should come from https://github.com/tigera/prometheus-docker/blob/7d311c6a8bfdf8496160488dcd6aac9ca7b7e915/Makefile#L16

Reply:
Changed to 2.45.0

Reply:
Changed to v2.45.0